playbooks/roles/local.defaults/templates/config.yml.j2

```diff
@@ -20,6 +20,54 @@ config:
 
   paths: {{ paths }}
 
+  accounts:
+  {%- for name in accounts | default({}) +%}
+    {{ name }}:
+      groups:
+    {%- for group in accounts[name].groups | default({}) +%}
+      {%- set acct = accounts[name].groups[group] +%}
+      {%- for idx in range(acct.instances | default(1)) +%}
+        {%- set group_name = group +%}
+        {%- if acct.instances is defined +%}
+          {%- set group_name = group ~ (idx + 1) +%}
+        {%- endif +%}
+        {{ group_name }}: {{ acct.gid + idx }}
```

Comment:
Shouldn't we record `gid` as separate entity to be processed as `<group_name>.gid` further during tasks from _create_groups.yml_?

Reply:
The groups don't require any additional information. In the settings file they are defined as an object with a `gid` field because we can also add the `instances` field, but in the config file we just need the number, so adding an object with a single field seems unnecessary. Instead of an object, I just use an integer.

In fact, the mistake is in _create_groups.yml_ where I use `gid` field when I shouldn't. I will fix that.

Reply:
Ok.